.pre-commit-config.yaml

```
@@ -40,7 +40,30 @@ repos:
     hooks:
       - id: regenerate-files
         name: regenerate generated files
-        language: system
+        language: python
```

I noticed that strings are quoted randomly in some places and not others. It's rather inconsistent. Perhaps, just don't quote them unnecessarily?

---

that's just me being bad at yaml and not knowing what's necessary or not :p
perhaps time for a yaml-formatter

---

Well, then you'd have to wrap literally **everything** in the file, including the keys:
```suggestion
        "language": "python"
```
Numbers like `3.11` are cast to float otherwise. And `3.10` caught many people by surprise back in the day (it's actually `3.1` unless quoted explicitly).

FWIW, I don't really like quotes. But if something needs escaping, I tend to use the following syntax (*especially* if there are quotes in the value that I don't want to bother about):
```suggestion
        language: >-
          python
```

---

see #3164, I'll do whatever for this PR and then it can be enforced later.